src/App.test.jsx

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable no-undef */
+  /* eslint-disable no-undef */
```

Comment:
Please resolve this eslint rule rather than ignoring it... if it's not affecting anything, take out this line

Reply:
This has been resolved, thank you.